packages/ui/src/json.ts

```diff
@@ -0,0 +1,186 @@
+/* @ts-nocheck */
+import components from '../src/importer'
+import apiOptions from './api-options'
+import { PropOptions } from 'vue-class-component'
+import en from '../../docs/src/locales/en/en.json'
+
+const path = require('path')
+const pack = require(path.join(__dirname, '../../package.json'))
+const fs = require('fs')
+
+type Prop = {
+  name: string,
+  type?: string | string[],
+  title: string
+  description?: string
+} & PropOptions
+
+type ComponentOption = {
+  name: string,
+  title: string
+  description?: string
+  type?: string | string[],
+}
+
+type Component = {
+  slots?: any[],
+  events?: any[],
+  methods?: any[],
+  props?: any[],
+  name: string
+  title: string
+  importIdentifier: string
+  description?: string
+}
+
+function splitCamelCase (input: string) {
+  return input.replace(/([a-z](?=[A-Z]))/g, '$1 ').toLowerCase()
+}
+
+function snakeToCamel (str: string) {

```

Comment:
Please no shortcuts, see [guideline](https://github.com/epicmaxco/vuestic-ui/blob/master/packages/docs/conventions.md#small-things-to-note).